bubblewrap.c

```diff
@@ -1458,6 +1485,9 @@ main (int argc,
   /* We want sigchild in the child */
   unblock_sigchild ();
 
+  if (label_exec(opt_exec_label) == -1)
```

Comment:
I doubt this would be an issue in practice though. I don't see any reason the app should have to contact its parent (other than possibly when debugging like @mrunalp said). In fact it should not even generally be aware of it, much like real apps don't care about pid1.